src/NuGet.Core/NuGet.Packaging/Signing/Utility/X509ChainHolder.cs

```diff
@@ -21,7 +22,9 @@ public sealed class X509ChainHolder : IDisposable
 
         public X509ChainHolder()
```

Comment:
Not at this time.  Adding a new parameter (e.g.:  `ILogger logger = null`) is a breaking change, and there isn't a strong use case for adding a new constructor.

When it's needed, it can be added.